qa-tests-backend/src/main/groovy/services/ImageService.groovy

```diff
@@ -35,20 +39,18 @@ class ImageService extends BaseService {
         getImageClient().invalidateScanAndRegistryCaches(EmptyOuterClass.Empty.newBuilder().build())
     }
 
-    static scanImage(String image, Boolean includeSnoozed = true, Boolean force = false) {
-        try {
-            def req = ImageServiceOuterClass.ScanImageRequest.newBuilder()
-                .setImageName(image)
-                .setIncludeSnoozed(includeSnoozed)
-                .setForce(force)
-                .build()
-            def response
-            withRetry(1, 15) {
-                response = getImageClient().scanImage(req)
-            }
-            return response
-        } catch (Exception e) {
-            log.error("Image failed to scan: ${image}", e)
+    static ImageOuterClass.Image scanImageNoRetry(String image, Boolean includeSnoozed = true, Boolean force = false) {
+        def req = ImageServiceOuterClass.ScanImageRequest.newBuilder()
+            .setImageName(image)
+            .setIncludeSnoozed(includeSnoozed)
+            .setForce(force)
+            .build()
+        return getImageClient().scanImage(req)
+    }
+
+    static ImageOuterClass.Image scanImage(String image, Boolean includeSnoozed = true, Boolean force = false) {
+        return evaluateWithRetry(10, 15) {
```

Comment:
Many of the CI flakes we've seen are related to the registry errors / network having issues (not ACS issues). 10x retries amongst many simultaneously running jobs could compound the issue.  The 1x retry was added not too long ago to help with these flakes - 1x was chosen with purpose.

I'm OK with increasing the number of retries but we should monitor and be prepared to revert it if we see increased failures as a result (such as our tenant becoming rate limited or exceeding quotas). Be good to measure how many scans failed on the first attempt but succeeded on a retry.